experimental/packages/otlp-transformer/src/metrics/types.ts

```diff
@@ -184,7 +184,7 @@ export interface IExponentialHistogramDataPoint {
   startTimeUnixNano?: number;
 
   /** ExponentialHistogramDataPoint timeUnixNano */
-  timeUnixNano?: string;
+  timeUnixNano?: number;
```

Comment:
Was this a mistake that got through before?

Reply:
This was an existing issue on main that predates my work on the exponential histogram.